extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -30,28 +30,50 @@ The example below contains a `bento-inline-gallery` consisting of three slides w
 ```html
 <head>
   <script src="https://cdn.ampproject.org/bento.js"></script>
-  <script
-    async
-    src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.js"
-  ></script>
-  <link
-    rel="stylesheet"
-    type="text/css"
-    href="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.css"
-  />
-</head>
+
+  <script async src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.js"></script>
+  <link rel="stylesheet" href="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.css">
+
+  <script async src="https://cdn.ampproject.org/v0/bento-base-carousel-1.0.js"></script>
+  <link rel="stylesheet" href="https://cdn.ampproject.org/v0/bento-base-carousel-1.0.css">
+
+  <style>
+    bento-base-carousel { height: 200px; }
+    bento-inline-gallery-thumbnails { height: 100px; }
+    bento-inline-gallery-pagination { height: 20px; }
+  </style>
 <body>
   <bento-inline-gallery id="inline-gallery">
-    <bento-inline-gallery-thumbnails
-      aspect-ratio="1.5"
-      loop
-    ></bento-inline-gallery-thumbnails>
-    <bento-base-carousel snap-align="center" visible-count="1.2" loop>
-      <img src="server.com/static/inline-examples/images/image1.jpg" />
-      <img src="server.com/static/inline-examples/images/image2.jpg" />
-      <img src="server.com/static/inline-examples/images/image3.jpg" />
+    <bento-inline-gallery-thumbnails loop></bento-inline-gallery-thumbnails>
+
+    <bento-base-carousel snap-align="center" visible-count="3" loop>
+      <img
+        src="https://images.unsplash.com/photo-1583511655857-d19b40a7a54e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1498&q=80"
```

Comment:
Can we use shorter URLs? Like `https://picsum.photos/200/300`. The browser will likely differentiate them if you add a unique query string to each.